extension/src/paymentHandler/get-origin-keys.handler.js

```diff
@@ -0,0 +1,18 @@
+const pU = require('./payment-utils')
+const c = require('../config/constants')
+const { getOriginKeys } = require('./web-component-service')
+
+async function handlePayment (paymentObject) {
```

Comment:
Maybe instead of `handlePayment` we can call it `execute` or just `handle` as it conflicts a bit with this: https://github.com/commercetools/commercetools-adyen-integration/pull/232/files#diff-b7da223fe1867c89db8963c55099bdccR17

Reply:
you're absolutely right!

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/232/commits/08c0812356cab3fb7d309798b7a5727c9fb204fa